### PR TITLE
Update custom_attributes.md

### PR DESCRIPTION
**Pull request by wendy-p**
Adding new operators for string and array filters

# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [x ] Yes (**Week of 10/17**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Data Infrastructure<br>Platform Infrastructure<br>Quality Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Internal Tools<br>Product Partnerships<br>SDKs<br>SMS</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Channels<br>FIX<br>Intelligence<br>Reporting<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Email (Composition and Infrastructure)<br>Ingestion<br>Messaging</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md [hunk 1]
```diff
@@ -101,6 +103,11 @@ String attributes are useful for storing user input, such as a favorite brand, a
 | Check if the string attribute **does not match** an inputted string| **DOES NOT EQUAL** | **STRING**<br>Not case sensitive.  |
 | Check if the string attribute **exists** on a user's profile | **IS NOT BLANK** | **N/A** |
 | Check if the string attribute **does not exist** on a user's profile | **BLANK** | **N/A** |
+| Check if the string exactly matches **any** of the inputted strings | **ANY OF** | **STRING**<br>Case sensitive; multiple strings allowed |
+| Check if the string attribute **does not exactly match any** of the inputted strings | **IS NOT ANY OF** | **STRING**<br>Case sensitive; multiple strings allowed |
+| Check if the string attribute **partially matches any** of the inputted strings | **CONTAINS ANY OF** | **STRING**<br>Case sensitive; multiple strings allowed |
+| Check if the string attribute **does not partially match any** of the inputted strings | **DOES NOT CONTAIN ANY OF** | **STRING**<br>Case sensitive; multiple strings allowed |
+
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md:110]:
```suggestion
```

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md [hunk 2]
```diff
@@ -93,6 +93,8 @@ Money spent should not be recorded by this method. Rather it should be recorded
 
 String attributes are useful for storing user input, such as a favorite brand, a phone number, or a last search string within your application. String attributes can be up to 256 characters long.
 
+Note: If you input any values with spaces in between, before, or after words, then Braze will also check for the same spaces.
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md:96]:
```suggestion
Note that if you input any values with spaces in between, before, or after words, then Braze will also check for the same spaces.
```

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md [hunk 3]
```diff
@@ -124,6 +131,8 @@ By default, the max length of an array for an attribute is set to 100. For examp
 
 If you'd like this maximum increased, reach out to your customer success manager. Your dashboard administrator can then increase the maximum length for individual arrays to over 100 from the **Custom Attributes** tab of the **Manage Settings** page. 
 
+Note: If you input any values with spaces in between, before, or after words, then Braze will also check for the same spaces.
```
Comment (KellieHawks) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md:134]:
```suggestion
Note that If you input any values with spaces in between, before, or after words, then Braze will also check for the same spaces.
```

**Review by KellieHawks — APPROVED** [review]

**Comment by wendy-p** [comment]
hey @KellieHawks I just realized a couple minor errors in the specific wording of the string filters (in the "Dropdown Filter" row for strings) - would you be able to make these changes, or is there a way I can go back and re-edit this?
- ANY OF --> should instead be IS ANY OF
- IS NOT ANY OF --> should instead be IS NONE OF
- CONTAINS ANY OF --> this is correct and does not need to be changed
- DOES NOT CONTAIN ANY OF --> should instead be DOESN'T CONTAIN ANY OF

**Comment by KellieHawks** [comment]
@wendy-p, I'll make sure it gets updated.

**Pull request merged**